### PR TITLE
Merge Test into Main

### PR DESCRIPTION
**Pull request by 9brian**
No changes have really been made, other than a point for discussion and the verification that the owners changes were able to merged with our code with no conflict.

### CODE REVIEW

File: apps/web/components/ui/WasabiRegionSelect.tsx
Comment (9brian) [inline apps/web/components/ui/WasabiRegionSelect.tsx:1]: For this region dropdown, is this still something we want to explore before the end of the pr? @nghiapham1026

File: apps/web/package.json
Comment (9brian) [inline apps/web/package.json:1]: I noticed that this is being imported and we were using this branch to test out a few bugs, but is this still needed? @nghiapham1026

File: apps/web/pages/new/wasabi.tsx
Comment (9brian) [inline apps/web/pages/new/wasabi.tsx:1]: If we decide not to go through with the dropdown, we can get rid of line 14

File: apps/web/pnpm-lock.yaml
Comment (9brian) [inline apps/web/pnpm-lock.yaml:1]: I'm not sure why this was changed, but it looks like it was importing dependencies? on line 4500 it was importing dotenv @nghiapham1026

File: apps/web/util/types.ts [hunk 1]
```diff
@@ -17,12 +17,12 @@ export type UploadingFile = {
...
-	git,
+  firebase,
+  s3,
+  minio,
+  backblaze,
+  deta,
+  git,
 	wasabi,
 	digitalocean,
   cloudflare,
```
Comment (sabrinaspage) [inline apps/web/util/types.ts:28]: why did we change this?

**Comment by sabrinaspage** [comment]
i mentioned in the channel but can we please follow this format for pull requests?

```
Overview
Please provide a general description of the changes for your reviewer(s).
Links
Ticket: Trello link with contextual information
Visual spec: Figma/Canva/Adobe link
Tech or product spec: Document of technical/product requirements
Attachments
Screenshot or recording of UI changes
Implementation Checklist
[x] Code changes have test coverage
ETC
```

i don't exactly understand what is happening in this PR

File: apps/web/hooks/useBucket.tsx [hunk 2]
```diff
@@ -25,22 +25,19 @@ export const ROOT_FOLDER: DriveFolder = {
...
+	const { keys } = useKeys();
+  
+	switch (Provider[keys.type] as Provider) {
+	  case Provider.firebase:
+		return useFirebase();
+	  case Provider.s3:
+	  case Provider.backblaze:
+	  case Provider.wasabi:
+	  case Provider.digitalocean:
+    case Provider.cloudflare:
```
Comment (sabrinaspage) [inline apps/web/hooks/useBucket.tsx:42]: why do we have CloudFlare in this PR? can we focus on digital ocean and wasabi before merging this?

**Review by sabrinaspage — CHANGES_REQUESTED** [review]

File: apps/web/util/types.ts (hunk 1 above)
Comment (9brian) [reply apps/web/util/types.ts:28]: This was after pulling the owners changes, it was reformatted.

File: apps/web/hooks/useBucket.tsx (hunk 2 above)
Comment (9brian) [reply apps/web/hooks/useBucket.tsx:42]: This was the owner's changes that were merged into the test branch

Comment (sabrinaspage) [reply apps/web/hooks/useBucket.tsx:42]: so this PR is a mix of you/nathan's changes and the owners? can we specifically merge the owner's changes into the repo (with or without a PR) and then open one for your changes?

According to [this article](https://fagnerbrack.com/one-pull-request-one-concern-e84a27dfe9f1#:~:text=While%20a%20commit%20can%20only,form%20a%20high%2Dlevel%20concern.&text=It%20is%20also%20important%20for%20a%20Pull%20Request%20to%20be%20atomic.), "A Pull Request represents a way to deliver value to the application in the form of a set of changes that together form a high-level concern."

if we are able to apply a separation of concerns to pull requests, that makes these changes make sense.

Comment (sabrinaspage) [reply apps/web/hooks/useBucket.tsx:42]: @9brian FYI: if you need to re-request me for a review, there is a button next to the reviewer's name that allows you to do so

Comment (sabrinaspage) [reply apps/web/hooks/useBucket.tsx:37]: like you're allowed to disagree with me but robust descriptions help engineers understand your POV

**Review by sabrinaspage — APPROVED** [review]

**Review by nghiapham1026 — APPROVED** [review]

**Pull request merged**